package.json

```diff
@@ -41,21 +41,21 @@
     "lodash.isequal": "4.5.0",
     "lodash.partition": "4.6.0",
     "lodash.truncate": "4.4.2",
-    "lucide-react": "0.402.0",
+    "lucide-react": "0.394.0",
     "papaparse": "5.4.1",
     "react": "18.3.1",
     "react-beautiful-dnd": "13.1.1",
     "react-csv": "2.2.2",
     "react-dom": "18.3.1",
     "react-ga4": "2.1.0",
-    "react-helmet-async": "2.0.5",
+    "react-helmet": "6.1.0",
```

Comment:
Looks like you reverted the changes from d42552798f2482fe8b1fc6f5aaf4f04ceda35b3d